sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/AzureMonitorSpanExporterProvider.java

```diff
@@ -0,0 +1,45 @@
+package com.azure.monitor.opentelemetry.exporter.implementation;
+
+import com.azure.monitor.opentelemetry.exporter.AzureMonitorExporterBuilder;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import io.opentelemetry.sdk.autoconfigure.spi.traces.ConfigurableSpanExporterProvider;
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+
+import java.util.Collection;
+
+public class AzureMonitorSpanExporterProvider implements ConfigurableSpanExporterProvider {
+    @Override
+    public SpanExporter createExporter(ConfigProperties configProperties) {
+        if (configProperties.getBoolean(AzureMonitorExporterProviderKeys.INTERNAL_USING_BUILDER, false)) {
+            return MarkerSpanExporter.INSTANCE;
+        }
+        return new AzureMonitorExporterBuilder().buildTraceExporter();
+    }
+
+    @Override
+    public String getName() {
+        return AzureMonitorExporterProviderKeys.EXPORTER_NAME;
+    }
+
+    public enum MarkerSpanExporter implements SpanExporter {
+
+        INSTANCE;
+
+        @Override
+        public CompletableResultCode export(Collection<SpanData> collection) {
+            return CompletableResultCode.ofSuccess();
```

Comment:
super.export

Reply:
there's no default interface method for `export`

Reply:
i see, just a placeholder. 👍